monkey/monkey_island/cc/services/telemetry/processing/scan.py

```diff
@@ -38,3 +43,11 @@ def update_edges_and_nodes_based_on_scan_telemetry(telemetry_json):
             )
         label = NodeService.get_label_for_endpoint(node["_id"])
         edge.update_label(node["_id"], label)
+
+
+def _machine_is_alive(machine_state: Mapping) -> bool:
```

Comment:
Consider renaming to `machine_found`. `_machine_is_alive` presumes that there is a machine on that IP and we're just checking if it's alive

Reply:
Isn't that kind of what we're doing, though? When you ping an IP, you presume that there's a machine there and determine whether or not it's alive based on the response.

Reply:
I'll use `host_responded`